src/api/v1/Audi/router.js

```diff
@@ -0,0 +1,43 @@
+const Router = require('express');
+const router = Router();
+
+const { authenticate } = require('../../../middlewares/authenticate');
+const { validate } = require('../../../middlewares/joivalidators');
+
+const {
+  audiSchema,
+  audiListSchema,
+  createSchema,
+  updateSchema,
+} = require('./validations');
+
+const {
+  getAudiById,
+  getAudiListByTheatreId,
+  createAudi,
+  updateAudi,
+} = require('./controller');
+
+router.post('/create', [
+  authenticate(1),
```

Comment:
instead of using values , kindly use global variables like USER_ROLE, USER_ADMIN,  (everywhere)